ecosystem/sep-0009.md

```diff
@@ -148,8 +148,34 @@ single multi-line string in the `address` field. This allows any address in the
 number of fields. If address parsing is necessary, parsing will be easier since the country, city, and postal code are
 already separate fields.
 
+## Card fields
+
+To pass card (such as credit card) details to the application, the client should pass card details via an object defined
+below. Note, that it's possible to either pass card details to the application, or the token, representing the card.
+This token may be fetched from a third-party source prior, for example, it can be a
+[Stripe token](https://stripe.com/docs/api/tokens/create_card), or any other service offering similar functionality may
+be used. When token is used, application should notify clients about the type of the token that it is expecting to
+receive. Usually, application would require either `token`, or set of: `number`, `expiration_date`, `cvc` and
+`holder_name` to be provided, but some applications may require extra fields.
+
+| Name                     | Type   | Description                                                                                 |
+|--------------------------| ------ | ------------------------------------------------------------------------------------------- |
+| `card.number`            | number | Card number                                                                                 |
+| `card.expiration_date`   | date   | Expiration month and year (e.g. `11-29`, November 2029)                                     |
```

Comment:
Lets be explicit about the formatting used. Is there an ISO standard for dates we can use?

Reply:
ISO 8601 define formats